client/src/components/KeyboardShortcutDialog.vue

```diff
@@ -13,25 +13,26 @@ export default defineComponent({
     shortcuts: [
       ['Next frame', [['→']]],
       ['Previous frame', [['←']]],
-      [
-        'Change slices',
-        [
-          ['q', 'w'],
-          ['a', 's'],
-          ['z', 'x'],
-        ],
-      ],
+      ['Next frame', [['↓']]],
+      ['Previous frame', [['↑']]],
```

Comment:
So right and down both go to the next frame, and left and up both go to the previous?